### PR TITLE
Feature/58 Add form validation and send converted form to BE

### PR DESCRIPTION
**Pull request by Kartm**

### CODE REVIEW

File: frontend-react/src/components/CreateEventForm/CreateEventForm.tsx [hunk 1]
```diff
@@ -117,59 +134,66 @@ export const CreateEventForm = ({onSubmit, owner}: CreateEventFormProps) => {
...
                     <Row gutter={16} justify="center">
                         <Col span={16}>
                             <Form.List name="guests" initialValue={[{email: '', obligatory: true}]}>
-                                {(fields, {add, remove}) => (
-                                    <>
+                                {(fields, { add, remove }) => (
+                                    <div>
                                         {fields.length === 0 && <Typography.Text>No guests added yet.</Typography.Text>}
-                                        <div style={{height: '25vh', overflowY: 'auto'}}>
+                                        <div style={{flex: 1, overflowY: 'hidden', height: '160px' }}>
```
Comment (Kaspr-whtev) [inline frontend-react/src/components/CreateEventForm/CreateEventForm.tsx:151]: Why are we going back to 'px'? Didn't we agree to convert everything to values relative to viewport (vh, vw)?

**Review by Kaspr-whtev — CHANGES_REQUESTED** [review]

File: frontend-react/src/components/CreateEventForm/CreateEventForm.tsx (hunk 1 above)
Comment (Kartm) [reply frontend-react/src/components/CreateEventForm/CreateEventForm.tsx:151]: I added it to make it look good for now. In later fixes we need to convert everything to vh

Comment (Kaspr-whtev) [reply frontend-react/src/components/CreateEventForm/CreateEventForm.tsx:151]: why not do it now?

Comment (Kaspr-whtev) [reply frontend-react/src/components/CreateEventForm/CreateEventForm.tsx:151]: because we will forget

Comment (Kartm) [reply frontend-react/src/components/CreateEventForm/CreateEventForm.tsx:151]: 1) I want main to be working ASAP
2) I want to do it in a separate task to make it easier to review, I'll add it to the board to not forget

Comment (Kaspr-whtev) [reply frontend-react/src/components/CreateEventForm/CreateEventForm.tsx:151]: ok then, but i feel like then this wont get done

**Review by Kaspr-whtev — APPROVED** [review]

**Pull request merged**